projects/storefrontlib/src/shared/components/anonymous-consents/dialog/form/anonymous-consent-form.component.scss

```diff
@@ -0,0 +1,209 @@
+/* Checkboxes should work without toggle button styling
```

Comment:
What is this?

Reply:
Miguel will take care of styles.

Reply:
Ok but this file shouldn't be here. Hope he removes it 😂

Reply:
🤞